rollingpin/deploy.py

```diff
@@ -88,12 +96,21 @@ def process_host(self, host, commands, timeout=0):
         try:
             log.info("connecting")
             connection = yield self.transport.connect_to(host.address)
-            for command in commands:
-                log.info(" ".join(command))
+            command_queue = commands[::-1]
```

Comment:
a little clearer to use `reversed(commands)`

Reply:
or really just `.pop(0)` -- the reversing is pretty confusing at first regardless of syntax!